config/checkstyle_non_main_files_suppressions.xml

```diff
@@ -17,6 +17,8 @@
              files="[\\/]test[\\/].*[\\/]asttreestringprinter[\\/]InputAstTreeStringPrinterFullOfSinglelineComments\.java"/>
   <suppress checks="NewlineAtEndOfFile"
              files="src[\\/]test[\\/]resources[\\/].*[\\/]InputNoCodeInFile6\.java"/>
+  <suppress checks="NewlineAtEndOfFile"
```

Comment:
This test case is needed to cover the `SINGLE_LINE_COMMENT` lexer rule's empty alternative.